mdoc/Mono.Documentation/Util/AssemblyLoader.cs

```diff
@@ -0,0 +1,607 @@
+﻿using Mono.Cecil;
```

Comment:
There's something strange about the changes in this file ... can you revert changes on AssemblyLoader to avoid adding unknown change noise?

Reply:
This class AssemblyLoader just for testing some special enum type only, I will remove it later. @anmeng10101 said maybe you want to know what I did lately so I open this draft PR for you but the issue has not been fixed.